rpc/endpoints.go

```diff
@@ -27,7 +28,7 @@ import (
 func StartIPCEndpoint(ipcEndpoint string, apis []API) (net.Listener, *Server, error) {
 	// Register all the APIs exposed by the services.
 	var (
-		handler    = NewServer()
+		handler    = NewServer(10, time.Duration(10)*time.Second)
```

Comment:
do we need it hard-coded?

Reply:
no i am checking whether this is causing ipc issues